lib/r10k/util/subprocess/runner/jruby.rb

```diff
@@ -14,8 +14,7 @@ def initialize(argv)
   end
 
   def run
-    spawn_opts = @cwd ? {:chdir => @cwd} : {}
-    stdout, stderr, status = Open3.capture3(*@argv, spawn_opts)
+    stdout, stderr, status = Open3.capture3(@argv.join(' '))
```

Comment:
This change will ignore the `cwd` variable, right? This looks like a regression on https://github.com/puppetlabs/r10k/commit/9b48cb27b71abe12646bae601e76f17f6c2d44c1 .